en-US/mozorg/internet-health/includes/podcast.lang

```diff
@@ -0,0 +1,20 @@
+## NOTE: Demo server is available at https://www-dev.allizom.org/internet-health/includes/podcast/
+## URL: https://www-dev.allizom.org/%LOCALE%/internet-health/includes/podcast/
```

Comment:
Include pages are not reachable, they're fragments of pages included into others.